Update userguide for Servlet 5

arjantijms opened the pull request:
I'm not sure if some of these changes need to be done locally in the servlet folder, or more globally in the Templates folder. If this is not correct, please advise.

cc @kwsutter 

Signed-off-by: arjantijms <arjan.tijms@gmail.com>

scottmarlow commented on user_guides/servlet/src/main/jbake/content/config.adoc:
```diff
@@ -58,7 +58,7 @@ with the necessary configuration steps for your implementation and place
 
 For more information, you may wish to view `<TS_HOME>/bin/xml/impl/glassfish/config.vi.xml`,
-the configuration file for Eclipse EE4J Jakarta EE 8 Compatible Implementation, Eclipse GlassFish.
+the configuration file for Eclipse EE4J Jakarta EE 9 Compatible Implementation, Eclipse GlassFish.
```
Should we remove ```Eclipse EE4J```?  

FYI, I see around 65 references to ```Eclipse EE4J``` in the jakartaee-tck source and 3 references to just ```EE4J```.

scottmarlow replied:
@arjantijms as per discussion on [Required Fixes on the naming of Jakarta EE across its mediums] It is NOT Eclipse Jakarta EE =/ It is Jakarta EE](https://www.eclipse.org/lists/jakarta.ee-community/msg02411.html), we should change "Eclipse EE4J Jakarta EE" to "Jakarta EE".

Could you please update your pull request to use "Jakarta EE 9".  Thank you!

Scott

arjantijms replied:
Thanks for the extra pair of eyes Scott, done!

arjantijms commented:
@edbratt 
>I believe CentOS 7 is the only OS tested

The version that the CI reports when I run the standalone TCK on the CI that has been given to our project is "Alpine Linux v3.12". See: https://ci.eclipse.org/servlet/job/2_servlet-run-tck-against-staged-build/lastSuccessfulBuild/artifact/servlet-tck/bin/summary.txt

arjantijms commented:
@edbratt I updated the issues you mentioned and found a lot more instances where javax.* and old terms like EJB where used, and update these.

Can you take a look again? Thx!

arjantijms commented on internal/docs/servlet/README.txt:
```diff
@@ -1,26 +1,26 @@
+3) Servlet TCK Version 5.0
 
-1. Download and install Java SE 7 software.
+1. Download and install Java SE 8 or Java 11 software.
```
I think I got this from another location in the Servlet files where this was stated, but I'll remove it to be sure. Thx!

alwin-joseph commented on install/servlet/docs/index.html:
```diff
@@ -38,30 +38,30 @@
 	</li>
 	<li>
-	  <p>The <a href="assertions/api-assertions.html">Javadoc Assertion List</a> lists all the javadoc assertions that are tested by the Jakarta Servlet TCK.</p></li>
+	  <p>The <a href="assertions/ServletSpecAssertions.html">Javadoc Assertion List</a> lists all the javadoc assertions that are tested by the Jakarta Servlet TCK.</p></li>
```
Did you mean ServletJavadocAssertions.html here

arjantijms replied:
Good catch, I'll correct them!

alwin-joseph commented on install/servlet/docs/index.html:
```diff
@@ -38,30 +38,30 @@
+	  <p>The <a href="assertions/ServletSpecAssertions.html">Javadoc Assertion List</a> lists all the javadoc assertions that are tested by the Jakarta Servlet TCK.</p></li>
 	<li>
-	  <p>The <a href="assertions/spec-assertions.html">Specification Assertion List</a> lists all the specification assertions that are tested by the Jakarta Servlet TCK.</p></li>
+	  <p>The <a href="assertions/ServletJavadocAssertions.html">Specification Assertion List</a> lists all the specification assertions that are tested by the Jakarta Servlet TCK.</p></li>
```
Did you mean ServletSpecAssertions.html here

scottmarlow commented:
Imo, this is definitely needed, I just don't want to merge it until we hear back from @arjantijms

edbratt commented:
@arjantijms The template mechanism was intended just to be a one-release stop-gap. But no effort was ever allocated to improving it since the plan was to distribute (refactor) production of these TCKs out to the API projects -- that hasn't happened either. I guess this is, once again proving: there is nothing more permanent than a temporary fix. Sorry for the frustration.

arjantijms commented:
> Imo, this is definitely needed, I just don't want to merge it until we hear back from @arjantijms and additional review comments are resolved.

I addressed the review comments. I think I can be merged now.

alwin-joseph commented:
> > Imo, this is definitely needed, I just don't want to merge it until we hear back from @arjantijms and additional review comments are resolved.
> 
> I addressed the review comments. I think I can be merged now.

Thanks Arjan
